### ISSUE TITLE
静态网站发布工具

### ISSUE THREAD

**Issue by egoist** [LightBulb/LightBulb#76]
国内版的 [surge.sh](https://surge.sh/) 并且能够配置使用自己的服务器。

**Comment by egoist** [comment]
ping @zorceta

**Comment by zorceta** [comment]
http://www.miibeian.gov.cn

**Comment by egoist** [comment]
说是国内其实使用 HK/JP 😂 Surge 用的 digitalocean 太慢了 🙃

**Comment by zorceta** [comment]
对 DO 印象不是很好，况且 [Linode 又疯了](https://blog.linode.com/2016/06/16/linodes-13th-birthday-gifts-for-all/)。
